src/transactions/TransactionEngineTest.java

```diff
@@ -0,0 +1,79 @@
+package transactions;
+import java.io.IOException;
+import java.util.Map;
+import java.io.File;
+
+/**
+ * TransactionEngineTest is used to test both the generateID() method in the Person
+ * class to ensure that a random 24 alpha numeric transaction ID is created for each person
+ * and also the getSchema() and getPersonMap() methods in the DataManager class
+ */
+public class TransactionEngineTest {
```

Comment:
This should extend from `junit.framework.TestCase`

Reply:
Should it be import junit.frameowrk.TestCase?

Reply:
Yeah, then you just extend TestCase